src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -361,6 +361,8 @@ private static bool OverloadResolutionResultIsValid<TMember>(ArrayBuilder<Member
 
             if ((options & Options.DynamicResolution) == 0)
             {
+                RemoveLowerPriorityMembers(results);
```

Comment:
nit: We're introducing more overload resolution features that are not supported by the dynamic binder. Is there a place we should record those divergences?

Reply:
Not as far as I'm aware.